packages/datagrid/src/containers/datagrid.container.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+
+import Component from '../components/';
+import DATAGRID_PROPTYPES from '../components/datagrid.proptypes';
+
+export default function DataGrid(props) {
+	const onFocusedCell = event => {
+		if (props.onFocusedCell) {
+			props.onFocusedCell(event);
+		}
+
+		if (get(props, 'actionCreators.onFocusedCell')) {
+			props.dispatchActionCreator(get(props, 'actionCreators.onFocusedCell'), event, {
+				props,
+			});
+		}
+	};
+
+	const onFocusedColumn = event => {
+		if (props.onFocusedColumn) {
+			props.onFocusedColumn(event);
+		}
+
+		if (get(props, 'actionCreators.onFocusedColumn')) {
+			props.dispatchActionCreator(get(props, 'actionCreators.onFocusedColumn'), event, {
+				props,
+			});
+		}
+	};
+
+	return (
+		<Component
+			{...props}
+			data={props.sourceData}
+			onFocusedCell={onFocusedCell}
+			onFocusedColumn={onFocusedColumn}
+		/>
+	);
+}
+
+DataGrid.propTypes = {
+	...DATAGRID_PROPTYPES,
+	sourceData: PropTypes.object,
+	actionCreators: PropTypes.shape({
+		onFocusedCell: PropTypes.string,
+		onFocusedColumn: PropTypes.string,
+	}),
```

Comment:
**Why do you name data `sourceData`?**

In connect, we should replace `props.data` with the real data, instead of adding `props.sourceData` + delete `props.data`

**Why do you extract actionCreators in a `props.actionCreators` ?**

You trigger the `props. onFocusedCell ` **and** the corresponding action creator but I don't think this is very useful.

IMO we should either : 
* have the same propTypes as the component, but adapt the `onFocusedCell` and `onFocusedColumn` in case they are strings
or
* have `onFocusedCellActionCreator` and `onFocusedColumnActionCreator` that the container change into `onFocusedCell` and `onFocusedColumn`

But in any case, the more we stay close to the component, the better.

Reply:
@jmfrancois wants keep this logic set in the subheaderbar (https://github.com/Talend/ui/blob/master/packages/containers/src/SubHeaderBar/SubHeaderBar.container.js#L64), trigger an event if the developer set a event on the container, it will be triggers with the actionCreator. Ok, i change to be closer that the component.

Reply:
What is the need ? Why would anyone set 2 event in 2 different ways ? :/
Ping @jmfrancois